tests/components/ResultsCount.test.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { Source, State } from '@yext/answers-headless-react';
+import { render, screen } from '@testing-library/react';
+import { ResultsCount } from '../../src/components/ResultsCount';
+import { mockAnswersState } from '../__utils__/mocks';
+
+
+const mockedStateUniversal: Partial<State> = {
+  searchStatus: {
+    isLoading: false
+  },
+  meta: {
+    searchType: 'universal'
+  },
+  universal: {
+    verticals: [{
+      appliedQueryFilters: [],
+      queryDurationMillis: 2,
+      results: [],
+      resultsCount: 2,
+      source: Source.Google,
+      verticalKey: 'string'
+    },
+
+
+    {
+      appliedQueryFilters: [],
+      queryDurationMillis: 2,
+      results: [],
+      resultsCount: 3,
+      source: Source.Google,
+      verticalKey: 'string'
+    }
+    ] }
+};
+
+const mockedStateVertical: Partial<State> = {
+  searchStatus: {
+    isLoading: false
+  },
+  meta: {
+    searchType: 'vertical'
+  },
+  vertical: {
+    resultsCount: 2
+  }
+};
+
+const mockedStateNoResult: Partial<State> = {
+  searchStatus: {
+    isLoading: false
+  },
+  meta: {
+    searchType: 'vertical'
+  },
+  vertical: {
+    resultsCount: 0
+  }
+};
+
+jest.mock('@yext/answers-headless-react');
+
+describe('ResultCounts', () => {
+  describe('ResultCountVertical', () => {
```

Comment:
nit: can we update the description to not be in camel case? maybe something like 'Results count for vertical results'. Similarly for the describe block for universal